### PR TITLE
[Datafactory] [Do Not Merge] Add interactive query API for Azure-VNet IR

### PR DESCRIPTION
**Pull request by ligengatms**
add interactive query API for Azure-VNet IR

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by ligengatms** [comment]
please do not merge for now, since we haven't finished backend service deploy yet

**Comment by sarangan12** [comment]
@ligengatms 

1. https://github.com/Azure/azure-rest-api-specs/pull/5036/checks?check_run_id=50061295
2. https://travis-ci.org/Azure/azure-rest-api-specs/jobs/479756231

Please fix the errors reported in the above links.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 1]
```diff
@@ -1269,6 +1269,99 @@
...
+        "tags": [
+          "integrationRuntimes"
+        ],
+        "operationId": "IntegrationRuntimes_EnableIntegrationRuntimeInteractiveQuery",
+        "x-ms-examples": {
+          "IntegrationRuntimes_EnableIntegrationRuntimeInteractiveQuery": {
+            "$ref": "./examples/IntegrationRuntimes_EnableInteractiveQuery.json"
+          }
+        },
+        "description": "Enable interactive query for an Auzre-VNet integration runtime.",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Auzre -> Azure

**Comment by sarangan12** [comment]
@ligengatms Ping

**Comment by hvermis** [comment]
Please do not merge this PR yet - we need to release Python SDK before getting any new changes in ADF swagger

**Comment by sarangan12** [comment]
@hvermis  Since there is no movement in this PR for more than a month, I am closing this. When you are ready to merge the changes, please fix the linter errors and reopen the PR. Thanks

**Pull request closed without merging**